qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -311,6 +312,7 @@ def setUp(self):
                             'qiita_prep_id': 1000,
                             'instrument_model': 'Illumina MiSeq',
                             'library_construction_protocol': 'AAAA',
+                            'insdc_nulls': 'NoT applicable',
```

Comment:
'NoT applicable' or 'Not applicable'?

Reply:
yup, just to make sure that it works with any casing ...

Reply:
If it's not too much trouble, would you mind adding a comment regarding the casing issue? I feel like it would be easy for someone in the future to see that and think 'quick fix' without thinking it through.

Reply:
IMOO is an overkill but if you feel strongly about it, I'll add it ...

Reply:
It is in a test method so I'll defer. I'm less likely to think it's unintentional in a test method.